src/opm/output/eclipse/Summary.cpp

```diff
@@ -2780,6 +2782,7 @@ eval(const int                          sim_step,
      const data::WellRates&             well_solution,
      const data::GroupAndNetworkValues& grp_nwrk_solution,
      GlobalProcessParameters&           single_values,
+     const Inplace&                     initial_inplace,
```

Comment:
This parameter is currently unused and will produce warnings.  Unless there is a follow-up PR that uses the values coming soon, I suggest you leave the object unnamed.

Reply:
It will be used in this PR.